### PR TITLE
Instantout: Add fee estimation to CLI

### PR DESCRIPTION
**Pull request by sputn1ck**
This pr adds a fee estimation to the instantout cli.

This is what it looks like:
image

### CODE REVIEW

File: swapserverrpc/instantout.proto [hunk 1]
```diff
@@ -130,6 +135,19 @@ message CancelInstantSwapRequest {
...
+    // The amount to swap in satoshis.
+    uint64 amount = 1;
+}
+
+message GetInstantOutQuoteResponse {
+    // The swap fee in satoshis.
+    uint64 swap_fee = 1;
+
+    // The pubkey to query the route to estimate offchain payment fees.
+    string pubkey = 2;
```
Comment (bhandras) [inline swapserverrpc/instantout.proto:148]: nit: can be updated now that pubkey is gone.

File: instantout/manager.go [hunk 2]
```diff
@@ -199,3 +202,59 @@ func (m *Manager) GetActiveInstantOut(swapHash lntypes.Hash) (*FSM, error) {
...
+		return Quote{}, err
+	}
+
+	dest, err := route.NewVertexFromStr(quoteRes.Pubkey)
+	if err != nil {
+		return Quote{}, err
+	}
+
+	// Get the offchain fee by calling the lnd client.
+	offChainFee, err := m.cfg.RouterClient.EstimateRouteFee(
```
Comment (bhandras) [inline instantout/manager.go:237]: nit: (just flagging), can be removed.

File: instantout/manager.go [hunk 3]
```diff
@@ -199,3 +201,50 @@ func (m *Manager) GetActiveInstantOut(swapHash lntypes.Hash) (*FSM, error) {
 
 	return fsm, nil
 }
+
+type Quote struct {
+	// ServiceFee is the fee in sat that is paid to loop service.
```
Comment (hieblmi) [inline instantout/manager.go:206]: nit: to the loop service.

File: loopd/swapclient_server.go [hunk 4]
```diff
@@ -1209,6 +1209,23 @@ func (s *swapClientServer) InstantOut(ctx context.Context,
 	return res, nil
 }
 
+func (s *swapClientServer) InstantOutQuote(ctx context.Context,
```
Comment (hieblmi) [inline loopd/swapclient_server.go:1212]: missing godoc

**Review by hieblmi — APPROVED** [review]
only nits, otherwise LGTM 🚀

**Pull request merged**